### DISCUSSION TITLE
Cannot Create Shipping Profile

### DISCUSSION THREAD

**Discussion by uzairfiaz-gelato** [etsy/open-api#665] [Q&A]
We are getting errors when creating a new Shipping profiles.
The used endpoint is: /v3/application/shops/{shop_id}/shipping-profiles.
In the response, we are getting this error `/object has validation errors: shipping_profile_name (name Test already exists)`. Although we do not get this profile in response of /v3/application/shops/{shop_id}/shipping-profiles API endpoint and we do not see any shipping profile with this name on the FE as well.

**Note:** Test is just used as an example here, the shipping profile name is the ones that we use and create.

Can someone help us to understand why this problem occurs? This happens from time to time not always.

**Comment by etsyachristensen** [comment]
In each shop, there can be only one shipping profile with any given name. We do not allow duplicate names for the same shipping profile. This also applies to profiles that have been deleted as we aren't able to fully delete the profiles for historical data concerns. They are soft deleted. Please try making the shipping profile names unique.

**Reply by uzairfiaz-gelato** [reply]
@etsyachristensen thanks for this, in my opinion, it should be mentioned somewhere in the documentation. Is there any way to get the soft deleted Shipping profiles?
